src/workspace-data/data-table/uri-viewer/UriPreview.js

```diff
@@ -76,6 +78,7 @@ export const UriPreview = ({ metadata, metadata: { uri, bucket, name }, googlePr
               [preview === null, () => 'Unable to load preview.'],
               [preview === undefined, () => 'Loading preview...'],
               [isImage(metadata), () => img({ src: preview, width: 400 })],
+              [canRender(metadata), () => h('object', { data: preview, type: metadata.contentType, width: 400, height: 400 })],
```

Comment:
An `<object>` element is used to render HTML and PDF files.

Reply:
I have [reached out to InfoSec](https://broadinstitute.slack.com/archives/CADU7L0SZ/p1724262580278949) to request review/approval for rendering `HTML` files.